src/main.rs

```diff
@@ -54,36 +50,22 @@ struct Args {
     #[clap(long, display_order = 4, default_value_t = 55555)]
     port: u64,
 
-    /// Number of iterations to run (skip) before measuring the latency.
-    #[clap(
-        long,
-        display_order = 5,
-        value_name = "ITERATIONS",
-        default_value_t = 10
-    )]
-    skip: u64,
+    /// Seconds to run (skip) before measuring the latency.
+    #[clap(long, display_order = 5, value_name = "DURATION", default_value_t = 0)]
+    skip_seconds: u64,
```

Comment:
This is optional, better declare as such.
```suggestion
    skip_seconds: Option<u64>,
```

Reply:
Done